packages/passport/wagmi-connector/src/index.ts

```diff
@@ -0,0 +1,71 @@
+import { passport } from '@imtbl/sdk';
+import { SwitchChainError, getAddress } from 'viem';
+import {
+  ChainNotConfiguredError, createConnector, normalizeChainId, CreateConnectorFn,
+} from '@wagmi/core';
+
+export function PassportConnector(pp: passport.Passport): CreateConnectorFn {
+  type Properties = {};
+
+  return createConnector<passport.Provider, Properties>((config) => ({
+    id: 'immutable-passport',
+    name: 'Immutable Passport',
+    type: 'Immutable Passport',
+
+    async setup() {
+      const provider = await this.getProvider();
+      provider.on('accountsChanged', this.onAccountsChanged);
+      provider.on('chainChanged', this.onChainChanged);
+      provider.on('disconnect', this.onDisconnect);
+    },
+    async connect() {
+      return {
+        accounts: await this.getAccounts(),
+        chainId: await this.getChainId(),
+      };
+    },
+    async disconnect() {
+      // TODO: is this implemented
```

Comment:
The `logout` function can be called on the root Passport module: https://github.com/immutable/ts-immutable-sdk/blob/ed9a7450c71f82e0f7e899201e5bf96749b501e2/packages/passport/sdk/src/Passport.ts#L225